tests/framework/gitlint_rules.py

```diff
@@ -8,20 +8,70 @@
 # pylint: disable=R0903
 
 
-class SignedOffBy(CommitRule):
-    """Make sure that each commit contains a "Signed-off-by" line."""
+class EndsSigned(CommitRule):
+    """Checks commit message body formatting.
+
+    Makes sure each commit message body ends with
+    1 or more signatures ("Signed-off-by"), followed by
+    0 or more co-authors ("Co-authored-by").
+    """
 
     # The name of the rule.
-    name = "body-requires-signed-off-by"
+    name = "body-requires-signature"
 
     # The unique id of the rule.
     id = "UC2"
 
     def validate(self, commit):
         """Validate user defined gitlint rules."""
-        for line in commit.message.body:
-            if line.startswith("Signed-off-by"):
-                return []
+        # Utilities
+        def rtn(stmt, i):
+            return [RuleViolation(self.id, stmt, None, i)]
```

Comment:
This was a lambda before, now `def`

E731 do not assign a lambda expression, use a def